test/parallel/test-tls-multiple-cas-as-string.js

```diff
@@ -30,6 +33,8 @@ function test(ca, next) {
   }
 }
 
+// `ca1` is not actually necessary for the certificate validation -- maybe
+// the fixtures should be written in a way that requires it?
```

Comment:
@nodejs/crypto If somebody knows how to do this, or that this does actually not make any sense, please raise your hand. :)